src/Symfony/Component/Form/ChoiceList/View/ChoiceGroupView.php

```diff
@@ -15,6 +15,8 @@
  * Represents a group of choices in templates.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @implements \IteratorAggregate<mixed, ChoiceGroupView|ChoiceView>
```

Comment:
```suggestion
 * @implements \IteratorAggregate<array-key, ChoiceGroupView|ChoiceView>
```
`mixed` seems strange for a key.

Based on https://github.com/symfony/symfony/blob/488a46f2b695f17b24599fef5e7d9713f445ac5e/src/Symfony/Component/Form/Tests/ChoiceList/Factory/DefaultChoiceListFactoryTest.php#L961-L989 

looks like it can be `int|string`